src/assets/js/markup-injectors/inject-update-books-list-on-dom.js

```diff
@@ -0,0 +1,17 @@
+import createBookItemMarkupTemplate from '../markup-templates/book-item-markup-template.js';
```

Comment:
* Nice work done so far 👏 but there's a little bug 🐛 I experience when I try to add books below 👇 is a recording of the said bug 

https://github.com/ITurres/Awesome-Books/assets/52497246/750ab263-487d-4a7e-bb68-a8a415270072

Reply:
Hi @Davidosky007! thank you for reviewing this PR 😃, What `error` did the console throw you?
My coding partner has experienced the same issue, However, this can be caused by the `Browser's localStorage settings`.
As described in the PR summary, this website has been tested on the following browsers **both with the LiveDemoLink and with LiveServer from the IDE**: 
- Brave
- Firefox
- Edge
- Chrome

I'll be waiting for your response 👨🏼‍💻 Hope you are having a good day!

Reply:
I tested it with my live server on my vscode

Reply:
@Davidosky007, It would be nice if you can let us know what is the **Error** that the `console` is throwing you! 🎯 

#### I here leave you a screen recording of the website being tested in the four Browsers mentioned above.
In the video I:
- Add new Book data
- Remove Existing Book data
- Reload the page to validate the permanence of the data in the `Browser` **localStorage**.

https://github.com/ITurres/Awesome-Books/assets/100724715/30b43b87-f697-4384-9585-0b3391794ee9